packages/cozy-scanner/src/CategoryGridItem.jsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import classNames from 'classnames'
+import palette from 'cozy-ui/transpiled/react/palette'
+import { Icon, IconStack } from 'cozy-ui/transpiled/react'
+
+import styles from './styles.styl'
+
+const CategoryGridItem = ({ isSelected, icon, theme, label }) => {
+  return (
+    <div
+      className={classNames(
+        styles['grid-item'],
+        'u-pb-half u-ph-half u-bxz  u-ellipsis u-c-pointer',
+        {
+          [styles['border-selected']]: isSelected,
+          [styles['grid-item__selected']]: isSelected && label,
+          [styles['grid-item__selected__without_label']]: isSelected && !label,
+          [styles['border-not-selected']]: !isSelected
```

Comment:
It seems a `GridItem` that would care about the `isSelected` prop could be extracted here.

It seems abnormal to have to put 4 different classes here also :/